content/docs/gto/get-started.md

```diff
@@ -0,0 +1,171 @@
+---
+description:
+  'Learn how you can use GTO to create Artifact Registry in Git repository'
+---
+
+# Get Started
+
+GTO helps you build an Artifact Registry on top of a Git repository (with a
+special case of Machine Learning Model Registry). You can register relevant
+versions of your files (e.g. ML model releases) and assign them to different
+deployment environments (testing, shadow, production, etc.). Git-native
+mechanisms are used, so you can automate the delivery of your ML project with
+CI/CD, and adopt a GitOps approach in general.
+
+This Get Started will walk you through basic GTO concepts and actions you would
+like to do in the Artifact Registry.
+
+## Showing the current state
+
+Assuming GTO is already [installed](/doc/gto/install) in your active Python
+environment, let's clone the example repo:
+
+```cli
+$ git clone https://github.com/iterative/example-gto
+$ cd example-gto
+```
+
+This repo represents a simple example of Machine Learning Model Registry. Let's
+review it:
+
+```cli
+$ gto show
+╒══════════╤══════════╤════════╤═════════╤════════════╕
+│ name     │ latest   │ #dev   │ #prod   │ #staging   │
+╞══════════╪══════════╪════════╪═════════╪════════════╡
+│ churn    │ v3.1.1   │ v3.1.1 │ v3.0.0  │ v3.1.0     │
+│ segment  │ v0.4.1   │ v0.4.1 │ -       │ -          │
+│ cv-class │ v0.1.13  │ -      │ -       │ -          │
+╘══════════╧══════════╧════════╧═════════╧════════════╛
+```
+
+Here we have 3 models: `churn`, `segment` and `cv-class`. The latest versions of
+them are shown in the column named `latest`. The latest is selected as the one
+having the greatest [SemVer](https://semver.org).
+
+Model versions could be promoted to different stages. Here we have 3 of them:
+`dev`, `prod` and `staging`. When a model was never promoted to a stage, we see
+`-` in the field.
+
+## Registering versions and assigning stages
+
+GTO can [register version](/doc/gto/command-reference/register) of artifacts and
+[assign stages to them](/doc/gto/command-reference/assign). Both functionalities
```

Comment:
Tip: remember we can add the GTO commands to auto-link (I forget where that file is, maybe in the theme repo). Then we can just

```suggestion
You can `gto register` artifacts and `gto assign` them to stages. Both functionalities
```

Reply:
@iterative/websites could you please help to understand what should I do to make this work? What file should I change for `gto assign` to link to a docs page about that command?

Reply:
The GTO setup seems different as the command reference is nested inside gto. 
So, we need a feature request to handle such a scenario.

Reply:
I have created a ticket https://github.com/iterative/gatsby-theme-iterative/issues/122. We will set priority on next week planning.